src/views/Pipeline.vue

```diff
@@ -117,6 +117,47 @@ export default defineComponent({
         event.target.complete();
       })
     },
+    async skipJob (job: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurance based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t('Dont skip'),
```

Comment:
Need to add apostrophe in Don't will need to fix internationalization file too.

```suggestion
              text: this.$t('Don't skip'),
```

Reply:
Added apostrophe in Don't